examples/textbooks/generate_openstax.py

```diff
@@ -21,6 +21,8 @@ class Args:
     """Number of TGI instances to use"""
     model_name: str = "mistralai/Mixtral-8x7B-Instruct-v0.1"
     """Model to prompt"""
+    prompts_dataset: str = "HuggingFaceTB/openstax_prompts_130k_v4"
```

Comment:
This is still private. Do we want to make them public?
<img width="752" alt="image" src="https://github.com/huggingface/llm-swarm/assets/5555347/cf1f57a6-9f10-4d83-8e5d-009ee4d3ec8f">

Reply:
I changed to cosmopedia datasets, which should become public in the release